tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -1,8 +1,8 @@
 # Known Limits
 # 1. Anchors are not supported in GHA
 # https://github.community/t/support-for-yaml-anchors/16128/90
-# 2. Nested Virutalizaiton isn't supported in Azure pipeline
-# https://developercommunity.visualstudio.com/t/enable-nested-virtualization-on-azure-pipelines/466384
+# 2. today most cloud-based CI services are still lacking hardware acceleration support from the host VM, 
+# which is the no.1 blocker for running tests on modern Android Emulators (especially on recent API levels) on CI.
 
```

Comment:
Is there a source for this info?

Reply:
https://ychescale9.medium.com/running-android-instrumented-tests-on-ci-from-bitrise-io-to-github-actions-a611e57ce71f

Reply:
https://github.com/ReactiveCircus/android-emulator-runner/
Pytorch uses it too.

Reply:
@[satyajandhyala](https://github.com/satyajandhyala) told me he successfully ran the emulator in Azure, and it worked fine.  Satya, would you mind sharing more details?

Reply:
@snnn @satyajandhyala any more comments